server/routes/index.js

```diff
@@ -1,13 +1,18 @@
 const express = require('express');
 const path = require('path');
-
 const router = express.Router();
-const voters = require('../model/voters');
+const voters = require("../model/voters");
 /* GET home page. */
+router.get("/", (req, res) => {
+  res.send("helloww");
+});
+router.get("/voters", (req, res) => {
 
-router.get('/voters', (req, res) => {
   res.json(voters);
 });
+router.post("/login", (req, res) => {
+  console.log(req.body);
```

Comment:
I'm assuming the user will be authenticated at some point here and this is not final, if so please change the log into a comment that'll you'll add authentication

Reply:
changed already, will be when  i push the next PR  ^_^